assets/js/components/Popup.js

```diff
@@ -260,8 +260,20 @@ module.exports = class Popup {
       top = topIfAbove;
     }
 
+    this.$link.parentNode.insertBefore(this.popupHitBox, this.$link);
     this.popupHitBox.style.left = `${left}px`;
     this.popupHitBox.style.top = `${top}px`;
+    this.popupHitBox.setAttribute('aria-expanded', 'true');
+    this.popupHitBox.setAttribute('tabindex', -1);
+    this.popupHitBox.setAttribute('aria-hidden', 'false');
+    this.popupHitBox.focus();
+  }
+
+  accessibilityPopupHitBox() {
```

Comment:
Could probably do with a more descriptive name for this method.

Refactor suggestion: with a better name you could combine the code in this method with the complementary code on [lines 266](https://github.com/elifesciences/pattern-library/pull/930/files#diff-752da1b00c0db6cd94aafc27aa81c946R266) - 268

For example, something like:
```
setPopupHitBoxAttributes(isToBeShown) {
  if (isToBeShown) {
    // do the showing stuff
  } else {
    // do the hiding stuff
  }
}
```
and then call the same function in the places where you need to change the attributes (equivalent to lines 266 etc and lines 340) with different arguments .

Bearing in mind the legacy sprawl in this class, I think this helps organise it that little bit more without costing too much effort.